modules/unsupported/vector-mosaic/src/main/java/org/geotools/vectormosaic/VectorMosaicFeatureSource.java

```diff
@@ -66,6 +67,9 @@
 /** FeatureSource for a vector mosaic. */
 public class VectorMosaicFeatureSource extends ContentFeatureSource {
     static final Logger LOGGER = Logging.getLogger(VectorMosaicFeatureSource.class);
+
```

Comment:
Add a note about what pattern this regex looks for with store names?

Reply:
Yes, done, and added the same in the GeoServer user docs.